core/src/test/java/org/infinispan/distribution/rehash/NonTxJoinerBecomingBackupOwnerTest.java

```diff
@@ -184,10 +184,10 @@ public boolean isSatisfied() throws Exception {
       });
 
       // Every ClusteredGetKeyValueCommand will be blocked before returning on cache0
-      CyclicBarrier beforeCache0Barrier = new CyclicBarrier(2);
-      BlockingInterceptor blockingInterceptor0 = new BlockingInterceptor(beforeCache0Barrier,
-            GetKeyValueCommand.class, false);
-      cache0.addInterceptorBefore(blockingInterceptor0, StateTransferInterceptor.class);
+//      CyclicBarrier beforeCache0Barrier = new CyclicBarrier(2);
```

Comment:
it can be removed. no need to keep it as comment. 
also, since you are touching this class, you could clean the unused imports :)